auth/src/main/res/values/colors.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="colorPrimary">#3F51B5</color>
-    <color name="colorPrimaryDark">#303F9F</color>
-    <color name="colorAccent">#FF4081</color>
+    <color name="fui_colorPrimary">#3F51B5</color>
```

Comment:
Uh oh, that's bad. I thought we said we shouldn't change this because of implementation details described [here](https://github.com/firebase/FirebaseUI-Android/issues/794).